adyenv6b2ccheckoutaddon/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -640,6 +677,14 @@ public void setAdyenPaymentServiceFactory(AdyenPaymentServiceFactory adyenPaymen
         this.adyenPaymentServiceFactory = adyenPaymentServiceFactory;
     }
 
+    public CommerceCartService getCommerceCartService() {
+        return commerceCartService;
+    }
+
+    public void setCommerceCartService(CommerceCartService commerceCartService) {
```

Comment:
As none of beans have required I just sticked to this pattern but generally I agree we should add @Required.